lib/sidekiq/cronitor.rb

```diff
@@ -33,14 +33,14 @@ def cronitor_disabled?(worker)
     end
 
     def job_key(worker)
-      worker.class.sidekiq_options.fetch(:cronitor_key, nil) ||
+      worker.class.sidekiq_options.fetch("cronitor_key", nil) ||
       options(worker).fetch(:key, worker.class.name)
     end
 
     def options(worker)
       # eventually we will delete this method of passing options
       # ultimately we want all cronitor options to be top level keys
-      opts = worker.class.sidekiq_options.fetch(:cronitor, {})
+      opts = worker.class.sidekiq_options.fetch("cronitor", {})
```

Comment:
can't believe I introduced this bug in my last branch 🤦‍♂️🤦‍♂️🤦‍♂️

specs caught it too and I thought the specs were broken, I am stupid.

this should fix my stupidity.